jira-macro/jira-macro-count/src/test/resources/macrojira2.test

```diff
@@ -0,0 +1,18 @@
+.runTransformations
+.#-----------------------------------------------------
+.input|xwiki/2.0
+.# Test the "count" style when used as a standalone macro.
+.#-----------------------------------------------------
+{{jiraCount url="http://localhost:8889" style="count" source="jql"}}
```

Comment:
There's a problem here. The jiraCount macro is not supposed to have `style` and `source` parameters

Reply:
Forgot to clean this. Fixed !